addons/point_of_sale/static/src/app/utils/input_popups/number_popup.xml

```diff
@@ -5,7 +5,8 @@
             <div t-if="props.subtitle" class="subtitle p-1 mx-auto" t-esc="props.subtitle"/>
             <div class="input-symbol pt-3">
                 <div class="popup-input value active form-control form-control-lg w-75 max-width-325px mx-auto">
-                    <span class="input-value" t-esc="props.formatDisplayedValue(this.state.buffer || '')"/>
+                    <span t-if="this.state.buffer || !props.placeholder" class="input-value" t-esc="props.formatDisplayedValue(this.state.buffer || '')"/>
+                    <small t-else="" class="text-muted" t-esc="props.placeholder"/>
```

Comment:
```suggestion
                    <span class="input-value" t-esc="props.formatDisplayedValue(this.state.buffer || props.placeholder || '')"/>
```

Reply:
The usage of the placeholder is to place a text, muted and smaller than the numbers shown in the input popup. That is why I put small and text-muted.